search-testing/src/main/java/no/unit/nva/indexing/testutils/Constants.java

```diff
@@ -1,12 +1,11 @@
 package no.unit.nva.indexing.testutils;
 
+@SuppressWarnings("PMD.ModifierOrder")
```

Comment:
you could just put it in the right order?

Reply:
or if you already have then i think this suppressor is redundant

Reply:
This is outside of scope, but pmd still complains.